waiter/waiter.go

```diff
@@ -0,0 +1,42 @@
+package waiter
+
+import (
+	"time"
+)
+
+var (
+	defaultPollInterval = 1 * time.Second
```

Comment:
I'd make it an exported `const`. It simplifies documentation considerably. And allows to do
```
NewWaiter(client, waiter.DefaultPollInterval)
```
Not thinking much about time.

Reply:
Make defaultPollInterval as const